src/Stratis.Bitcoin.Features.Consensus/Rules/IConsensusRules.cs

```diff
@@ -48,6 +48,11 @@ public interface IRuleRegistration
         /// The rules that will be registered with the rules engine.
         /// </summary>
         /// <returns>A list of rules.</returns>
+        /// <remarks>
+        /// It is important to not that there is high importance to the order the rules are registered 
```

Comment:
"to note"

Reply:
Ah thanks

Reply:
@dangershony this still needs to be fixed